scss/tech.scss

```diff
@@ -11,31 +7,33 @@
   margin-bottom: 15px;
 }
 
-.tech {
-  @extend %tech-border-radius;
+%tech {
   @extend %size-tech;
   align-items: center;
-  background-color: $earls-green;
   display: flex;
   flex-direction: column;
-  left: 45px;
   margin: 15px 15px;
   position: relative;
   padding: 0 25px 0 5px;
+}
+
+.tech {
+  @extend %tech;
```

Comment:
why do you extend tech from %tech?... you can do this in html like `first-tech tech`